src/terra-dev-site/example/ResponsiveListAndTableExample.scss

```diff
@@ -0,0 +1,43 @@
+:local {
+  .table {
+    table-layout: fixed;
+  }
+
+  .start-accessory-column {
+    width: 40px;
+  }
+
+  .content-accessory.content-accessory-align-top {
+    vertical-align: top;
+  }
+
+  .content-accessory.content-accessory-align-center {
+    vertical-align: middle;
+  }
+
+  .content-accessory.content-end-accessory {
+    text-align: right;
+  }
+
+  .content-accessory,
+  .content-comment,
+  .content-display {
+    word-wrap: break-word;
+  }
+
+  .content-display {
+    flex: 1 1 0;
+    min-width: 75px;
+    padding-right: 1rem;
+  }
+
+  .content-accessory {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
```

Comment:
Remove this. Top align the icons and give it a height of 20px to align it with the user icon in the next cell.

Reply:
Removed and added padding-top to better line up the icons
<img width="1078" alt="Screen Shot 2021-01-04 at 3 55 43 PM" src="https://user-images.githubusercontent.com/39568848/103583507-6cfca880-4ea5-11eb-9fd5-c8321c18b47b.png">

https://github.com/cerner/terra-ui/pull/310/commits/353727590506d9c58f0f44b2cbfd34a9accec9b7#diff-9d259ec4f7b0e6e24de37dbf10bcc38de4494081ed2b22e2fc4b9a37f2c66879L37